starthost-plugin/src/main/java/br/com/autonomiccs/starthost/systemVm/AutonomiccsStartHostSystemVmManager.java

```diff
@@ -357,24 +352,7 @@ public void afterPropertiesSet() throws Exception {
     }
 
     private void copyStartHostApplicationToTempFile() throws FileNotFoundException, IOException {
-        String cloudStackWebappLibFolder = getCloudStackWebappLibFolder();
-        String startHostApplicationFullQualifiedPath = cloudStackWebappLibFolder + "wakeonlan-service-" + wakeOnLanHostApplicationVersion;
-        FileInputStream startHostApplicationStream = new FileInputStream(startHostApplicationFullQualifiedPath);
-        startHostApplicationExecutable = createTempFileForStream(startHostApplicationStream);
-        IOUtils.closeQuietly(startHostApplicationStream);
-    }
-
-    private String getCloudStackWebappLibFolder() {
-        String regexFindCloudStackLibFolder = "jar:file:(.+)\\/.*\\.jar.*";
-        Pattern patternFindCloudStackLibFolder = Pattern.compile(regexFindCloudStackLibFolder);
-
-        String fileUsedToUncoverCloudStackLibFolder = "/application.yml";
-        URL resource = getClass().getResource(fileUsedToUncoverCloudStackLibFolder);
-        Matcher matcher = patternFindCloudStackLibFolder.matcher(ObjectUtils.toString(resource));
-        if(matcher.find()){
-            return matcher.group(1) + "/";
-        }
-        throw new CloudRuntimeException("Could not find cloudstack lib folder.");
+    	startHostApplicationExecutable = AutonomiccsJarAbstraction.WAKEONLAN.getJar();
```

Comment:
Isn't this information being used to load things that are not the .jar??

On Thu, Jun 2, 2016 at 8:49 PM, Rafael Weingärtner <notifications@github.com
> wrote:

> In
> starthost-plugin/src/main/java/br/com/autonomiccs/starthost/systemVm/AutonomiccsStartHostSystemVmManager.java
> <https://github.com/Autonomiccs/autonomiccs-platform/pull/23#discussion_r65638319>
> :
>
> > -        startHostApplicationExecutable = createTempFileForStream(startHostApplicationStream);
> > -        IOUtils.closeQuietly(startHostApplicationStream);
> > -    }
> > -
> > -    private String getCloudStackWebappLibFolder() {
> > -        String regexFindCloudStackLibFolder = "jar:file:(.+)\\/.*\\.jar.*";
> > -        Pattern patternFindCloudStackLibFolder = Pattern.compile(regexFindCloudStackLibFolder);
> > -
> > -        String fileUsedToUncoverCloudStackLibFolder = "/application.yml";
> > -        URL resource = getClass().getResource(fileUsedToUncoverCloudStackLibFolder);
> > -        Matcher matcher = patternFindCloudStackLibFolder.matcher(ObjectUtils.toString(resource));
> > -        if(matcher.find()){
> > -            return matcher.group(1) + "/";
> > -        }
> > -        throw new CloudRuntimeException("Could not find cloudstack lib folder.");
> > +    	startHostApplicationExecutable = AutonomiccsJarAbstraction.WAKEONLAN.getJar();
>
> now that you are no longer using "wakeOnLanHostApplicationVersion"
> variable, it should be removed from the
> "AutonomiccsStartHostSystemVmManager" object. Also, do not foger to remove
> the "wakeOnLanHostApplicationVersion" object that is created in the
> applicationContext, the file is "spring-autonomiccsPlugins-context.xml"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Autonomiccs/autonomiccs-platform/pull/23/files/1731fb3872fb4d5665e5dbc1b9cba2530f3e4a04#r65638319>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKiipP5fontIPuvQyjrBLj2F-Uu32N46ks5qH2wJgaJpZM4Is-5q>
> .
>